public/app.css

```diff
@@ -1,46 +1,43 @@
-* {

```

Comment:
Should we keep this, or was it a previous override for react-static that wasn't needed any more?

Reply:
I suspect it is coming from another project, this website we should only need this to be applied on `html, body`.